mapzen/whosonfirst/spatial/sqlite.py

```diff
@@ -0,0 +1,111 @@
+# THIS IS WET PAINT AND PROBABLY DOESN'T WORK YET
```

Comment:
Is this still the case? ;-)

Reply:
it _works_ - I'll test a bit more and add a commit to update that comment..